src/SkyImages.jl

```diff
@@ -449,86 +449,6 @@ world_to_pix{T <: Number}(patch::SkyPatch, world_loc::Vector{T}) =
 
 
 """
-A fast function to determine which sources might belong to which tiles.
-
-Args:
-  - tiles: A TiledImage
-  - patches: A vector of patches (e.g. for a particular band)
-
-Returns:
-  - An array (over tiles) of a vector of candidate
-    source patches.  If a patch is a candidate, it may be within the patch radius
-    of a point in the tile, though it might not.
-"""
-function local_source_candidates(
-    tiles::TiledImage, patches::Vector{SkyPatch})
-
-  # Get the largest size of the pixel ellipse defined by the patch
-  # world coordinate circle.
-  patch_pixel_radii =
-    Float64[patches[s].radius * maximum(abs(eig(patches[s].wcs_jacobian)[1]))
-            for s=1:length(patches)];
```

Comment:
In the new version of `local_source_candidates` in `ModelInit`, we no longer call `eig`, so hopefully this should be a performance bump.